### ISSUE TITLE
Can I introduce JuliaQuantum on the meetup@Shenzhen

### ISSUE THREAD

**Issue by Roger-luo** [JuliaQuantum/JuliaQuantum.github.io#37]
Hi all,

The [Chinese Julia community](http://julialang.cn/) is going to hold a meet-up at Shenzhen . May I introduce the org during the meet-up? I'll upload my slide later on github, if you would like me to introduce this org on the meetup.

**Comment by Roger-luo** [comment]
@amitjamadagni  My slides are here, [slides@rogerluo.cc](http://rogerluo.cc/slides/), I chat with a guy from Tencent, and I'll redesign the package Baozi.jl to make it pluggable :-)

**Comment by amitjamadagni** [comment]
@Roger-luo I just went through the slides ! Looks nice though they are some styling elements in it ... I do not know if they should be that way or they can be corrected ! Anyway thanks for the share, you could add it to the Resources once the meetup is done :+1:

**Comment by Roger-luo** [comment]
@amitjamadagni The meetup is done :-) I'll make it into Resources after I'm back to campus~

For the styling elements,The slides was generated by markdown base on my project: [Baozi.jl](https://github.com/Roger-luo/Baozi.jl), however this project have a lot of bugs.....I discussed with a frontend engineer today, and I'm going to rewrite it into a metalsmith-like structure, which could be easier and lighter for users. And I'll fix these bugs after I rework the structure....

The Baozi.jl was a rush for preparing the meetup's slides, as I hope there could be a tool in pure Julia for generating slides and static htmls...

**Comment by i2000s** [comment]
> The Baozi.jl was a rush for preparing the meetup's slides, as I hope there could be a tool in pure Julia for generating slides and static htmls...

Have you tried Rknit with Julia engine? It can produce slides in Beamer, HTML or other formats. I think there are tons of tex editors for this purpose, so that you can save time to do something more productive and innovative. 

Besides, I would suggest you to get used to post News on the [JuliaQuantum frontpage](http://juliaquantum.github.io) to meetups and your recent progress of JuliaQuantum projects. There are only 16 watchers of this repo, but there are tons of people checking our website regularly. 

Bests,
Qi

**Comment by Roger-luo** [comment]
@i2000s Rknit is quite inspiring, but it seems it is different from what Baozi.jl aimed to do (a static site generator like jekyll). It can be used to generate slides, just because I used it to organize revealjs. The Baozi.jl project will be designed to allow users (not only scientists) to be plugable like [metalsmith](http://www.metalsmith.io/), which also inspired me in designing the structure of QuCmp.jl.

I have been thinking about making QuCmp.jl become a "plugable" framework like the markdown parser in Base.Markdown, which may solve the problem that different computing models could have a different interface and makes it hard to use. But this can only be tried after I finished the one-way part. So long way to go...

And for the post, do I need to ask you check it or just merge it myself?

**Comment by i2000s** [comment]
@Roger-luo It would be good to make a PR and have @amitjamadagni reviewing it, since he is also preparing a poster. 

BTW, I will follow up on the summary of the GSoC application process. An email will be sent to you later, and hopefully we can learn something helpful for the future events. 

Thanks,
Qi

**Comment by amitjamadagni** [comment]
Sorry for the delay in reviewing, though this PR https://github.com/JuliaQuantum/JuliaQuantum.github.io/pull/38 looks ready I guess it would be better if we could have an article in the news section explaining more about the new package and the recent presentation at the meetup ! We could add it to the projects once we have a bit more functionality in the master of QuCmp.jl (sorry once again though @Roger-luo has added his code, it is taking time for me to review, I will provide my thoughts as soon as possible, and also request other members to chip in with the thoughts on the open PR's in the newly created repo) ... @i2000s Please do let me know the above mentioned PR could go in right away !

**Comment by amitjamadagni** [comment]
I got confused between the open PR #38 and the post in the news section that is being discussed :-( @Roger-luo I guess you could write up an article mentioning the aim of new package and add your thoughts on the meetup, this will go into the news section of the site.

Also regarding PR #38 I guess we could wait until we have some functionality merged into the master for #38 to be merged. I guess providing a link in the above article will direct to the project and once we have something to deliver via the new package we could add it up to the projects list on the site. Please do correct me if waiting for #38 is does not sound like a good idea !

**Comment by Roger-luo** [comment]
@amitjamadagni Uh,yes. So should I add a news in `news` folder like 'New repository added: QuCmp' and write an article to introduce it? or which folder would be more appropriate? Thanks for the reviewing. And how about moving this conversation to PR #38 ?

**Comment by amitjamadagni** [comment]
@Roger-luo I guess #38 looks good but we can merge it later. Please add the article to https://github.com/JuliaQuantum/JuliaQuantum.github.io/tree/master/news/_posts by creating a new PR. Also you can compose the new post in two parts one introducing the packaging and other about the meetup with some appropriate title !
